client/me/help/help-contact/index.jsx

```diff
@@ -666,7 +666,7 @@ const HelpContact = React.createClass( {
 				<Card className={ this.canShowChatbox() ? 'help-contact__chat-form' : 'help-contact__form' }>
 					{ this.getView() }
 				</Card>
-				<HappychatConnection />
+				{ this.props.shouldStartHappychatConnection && <HappychatConnection /> }
```

Comment:
I agree that we need a better Happychat entry point for that I created 750-gh-happychat.

> If this solves an immediate problem without too much hassle, that's fine — but I just wanted to raise this as another issue that might keep biting us.

Indeed it does as JPOP have an ongoing experiment and having proper stats about jpop chats would be helpful.